db/languages.yml

```
@@ -7,22 +7,28 @@ c++:
   :lexer: c++
   :compiled: yes
   :interpreted: no
-  :current: c++11
+  :current: c++17
   :extension: .cpp
   :source_filename: program
   :exe_extension: .exe
   :processes: 1
   :variants:
-    c++03:
-      :name: "C++03"
+    c++03:  # depreciated
```

"Depreciate" is an accounting term. The word you're looking for is "deprecate" with no "i".

```suggestion
    c++03:  # deprecated
```

---

Good spot! My legendary spelling and grammar skills at it again.